syntax = "proto3";

package yir.main;

option go_package = "yir/api;main";

message Tirads {
  double tirads_23 = 1 ;
  double tirads_4 = 2 ;
  double tirads_5 = 3 ;
}

message KafkaSegment {
  string id = 100;
  string formation_id = 200;
  string image_id = 300;

  Tirads tirads = 500;
}

message KafkaFormation {
  string id = 100;

  Tirads tirads = 200;
  bool ai = 300;
}

message uziProcessed {
  repeated KafkaFormation formations = 100;
  repeated KafkaSegment segments = 200;
}

message uziSplitted {
  string uzi_id = 100;
  repeated string pages_id = 200;
}

